draft-ietf-rtcweb-jsep.xml

```diff
@@ -723,18 +723,26 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
               candidate gathering and maximize multiplexing, at the cost of less
               compatibility with legacy endpoints.
               </t>
-              <t></t>
-              <t hangText="max-bundle-and-rtcp-mux:">Similar to max-bundle,
-              but RTCP candidates are not gathered. This policy reduces the
-              candidates that must be gathered to the absolute minimum,
-              but will not be compatible with legacy endpoints that do not
-              support RTCP mux.
-              </t>
             </list>
             As it provides the best tradeoff between performance and
             compatibility with legacy endpoints, the default BUNDLE policy
             MUST be set to "balanced".
           </t>
+
+          <t>The
+          application can specify its preferred policy regarding use of RTP/RTCP
+          multiplexing <xref target="RFC5761"/> using one of the following policies:
+          <list style="hanging">
+            <t hangText="max-compat:">The browser will gather both RTP and
```

Comment:
I see what you were trying to do here, but I agree it seems silly. After thinking about it a bit, I think it's a distinct animal from the bundle policy. How about "negotiate" and "require" as policies?